hook-tests/014-set-motd.test

```diff
@@ -7,3 +7,5 @@ grep -q "This Ubuntu Core 18 machine is a tiny" etc/motd
 test ! -e etc/update-motd.d/10-help-text
 test ! -e lib/systemd/system/motd-news.service
 test ! -e lib/systemd/system/motd-news.timer
+
+grep -q "ENABLE=0" etc/default/motd-news
```

Comment:
This should still be ENABLED=0, let's not change the variable name

Reply:
Absolutely right, will fix it.